src/registrar/management/commands/copy_names_from_contacts_to_users.py

```diff
@@ -0,0 +1,236 @@
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = """Copy first and last names from a contact to
+    a related user if it exists and if its first and last name
+    properties are null or blank strings."""
+
+    # ======================================================
+    # ===================== ARGUMENTS  =====================
+    # ======================================================
+    def add_arguments(self, parser):
+        parser.add_argument("--debug", action=argparse.BooleanOptionalAction)
+
+    # ======================================================
+    # ===================== PRINTING  ======================
+    # ======================================================
+    def print_debug_mode_statements(self, debug_on: bool):
+        """Prints additional terminal statements to indicate if --debug
+        or --limitParse are in use"""
+        TerminalHelper.print_conditional(
+            debug_on,
+            f"""{TerminalColors.OKCYAN}
+            ----------DEBUG MODE ON----------
+            Detailed print statements activated.
+            {TerminalColors.ENDC}
+            """,
+        )
+
+    def print_summary_of_findings(
+        self,
+        skipped_contacts,
+        eligible_users,
+        processed_users,
+        debug_on,
+    ):
+        """Prints to terminal a summary of findings from
+        copying first and last names from contacts to users"""
+
+        total_eligible_users = len(eligible_users)
+        total_skipped_contacts = len(skipped_contacts)
+        total_processed_users = len(processed_users)
+
+        logger.info(
+            f"""{TerminalColors.OKGREEN}
+            ============= FINISHED ===============
+            Skipped {total_skipped_contacts} contacts
+            Found {total_eligible_users} users linked to contacts
+            Processed {total_processed_users} users
+            {TerminalColors.ENDC}
+            """  # noqa
+        )
+
+        # DEBUG:
+        TerminalHelper.print_conditional(
+            debug_on,
+            f"""{TerminalColors.YELLOW}
+            ======= DEBUG OUTPUT =======
+            Users who have a linked contact:
+            {eligible_users}
+
+            Processed users (users who have a linked contact and a missing first or last name):
+            {processed_users}
+
+            ===== SKIPPED CONTACTS =====
+            {skipped_contacts}
+
+            {TerminalColors.ENDC}
+            """,
+        )
+
+    # ======================================================
+    # ===================    USER    =====================
+    # ======================================================
+    def update_user(self, contact: Contact, debug_on: bool):
+        """Given a contact with a first_name and last_name, find & update an existing
+        corresponding user if her first_name and last_name are null.
+
+        Returns the corresponding User object.
+        """
```

Comment:
(Nice to have) Could you note in the function description that this returns a tuple than just a user object?